src/components/Tag/Tag.scss

```diff
@@ -40,11 +38,11 @@ $icon-size: rem(16px);
   margin-left: spacing(extra-tight);
   border-radius: 0 border-radius() border-radius() 0;
 
+  // Start - Delete below me during global theming rollout
   &:hover,
   &:focus,
   &:active {
     @include recolor-icon(color('ink'));
-    background: color('sky', 'dark');
```

Comment:
We'll need to add this line back in, as this will be a visual regression without global theming enabled.